.lesshst

```diff
@@ -0,0 +1,2 @@
+.less-history-file:
```

Comment:
@ttseng does this need to go in gitignore?

Reply:
Pretty sure it can be removed altogether so I got rid of it.